plugin-server/src/main/ingestion-queues/session-recording/session-recordings-blob-consumer.ts

```diff
@@ -360,11 +364,11 @@ export class SessionRecordingBlobIngester {
         this.sessions = new Map()
     }
 
-    private async destroySessions(sessionsToDestroy: SessionManager[]): Promise<void> {
+    async destroySessions(sessionsToDestroy: SessionManager[]): Promise<void> {
         const destroyPromises: Promise<void>[] = []
 
         sessionsToDestroy.forEach((sessionManager) => {
-            this.sessions.delete(sessionManager.sessionId)
+            this.sessions.delete(sessionManagerKey(sessionManager.teamId, sessionManager.sessionId))
```

Comment:
😮 this was a rooky mistake 🙈

Reply:
Maybe better would be to iterate over it with .entries or something so we have the key and the value instead of rebuilding the key which might be prone to some other error in the future?